examples/tests/hu.elte.txtuml.examples.machine/src/machine1/x/XMachine1Configuration.java

```diff
@@ -0,0 +1,10 @@
+package machine1.x;
+
+import hu.elte.txtuml.api.deployment.Group;
+import machine1.x.model.Machine;
+import machine1.x.model.User;
+
+@Group(contains = { Machine.class, User.class })
+public class XMachine1Configuration {
```

Comment:
I think, the Machine can be deleted, but the J contains az extra class which help me testing the generated c++ code easily.